pipelines/broad/arrays/single_sample/test_inputs/Scientific/204126290052_R01C01_NA12878_3.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Should this be added to the readme?

Reply:
Yes!

Reply:
I've added that to the readme (or actually, removed '_3', and updated '_2')